validator/test/fixtures/constraints/sql_allowed_authorized_networks_whitelist/data.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: constraints.gatekeeper.sh/v1alpha1
+kind: GCPSQLAllowedAuthorizedNetworksConstraintV1
+metadata:
+  name: sql_allowed_authorized_networks_whitelist
+spec:
+  severity: high
+  match:
+    gcp:
+      target: ["organization/*"]
+  parameters:
+    authorized_networks: ["35.187.118.0/24"]
```

Comment:
can we add a test to make sure that subnetwork also passes? for example, change authorized_networks parameter to ["35.187.118.0/20"] should still work.

Reply:
Added a new test.